containers/dapr-dotnetcore-3.0/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+version: '3.5'
+services:
+  docker-in-docker:
+    # Uncomment the next line to use a non-root user. On Linux, this will prevent
```

Comment:
This comment is a bit out of date since we now have a new `remoteUser` property in v1.41 that just shipped.  That said, it's easy enough to update post-merge.

Reply:
I've updated the comments to more closely match the existing docker-in-docker template w.r.t. `remoteUser`.